lib/src/models/spi_bfm/spi_main_driver.dart

```diff
@@ -38,7 +38,7 @@ class SpiMainDriver extends PendingClockedDriver<SpiPacket> {
     unawaited(super.run(phase));
 
     Simulator.injectAction(() {
-      intf.csb.put(1);
+      intf.csb[0].put(1);
```

Comment:
this is incomplete on the BFM side, right?  we need ways to drive different CSB's, not just always 0